CHANGELOG.md

```diff
@@ -21,6 +21,16 @@ The format is based on [Keep a Changelog].
   option now ([#305]).
 * The special text property `selectrum-candidate-full` to change the
   canonical representation of a candidate has been removed ([#403]).
+* The `selectrum-read` API has been deprecated and made private. The
+  intention of this change is to encourage users instead to rely on
+  the plain `completing-read` function, which is completion system
+  agnostic ([#446]).
```

Comment:
@minad I forgot the deprecations go under their own headings (they are not breaking changes yet) I will do that before the release then.

Reply:
Okay, I am sorry for that.

Reply:
No problem at all. It is really nice that selectrum-read is deprecated now!